gratipay/security/__init__.py

```diff
@@ -52,7 +52,8 @@ def add_headers_to_response(website, response):
         response.headers['content-security-policy-report-only'] = (
             "default-src 'self';"
             "script-src 'self' assets.gratipay.com 'unsafe-inline';"
-            "style-src 'self' assets.gratipay.com downloads.gratipay.com cloud.typography.com;"
+            "style-src 'self' assets.gratipay.com downloads.gratipay.com cloud.typography.com"
+            "          'sha256-WLocK7HeCKzQLS0M+PGS++5IhyfFsOA5N4ZCeTcltoo=';"
```

Comment:
It's a hash of the [`<style>`](https://github.com/gratipay/gratipay.com/blob/2097/www/~/%25username/widget.html.spt#L12-L104), which Google Chrome shows me in the developer console as part of its CSP error message. I copied and pasted it from there. It will need to be updated anytime that content changes (it hasn't [changed](https://github.com/gratipay/gratipay.com/commits/2097/www/~/%25username/widget.html.spt) in over two years). Commented in b5390d96d3d43806c5e4ccd0d116fcf5dac26108.